bin/hexdump

```diff
@@ -1,5 +1,19 @@
 #!/usr/bin/perl
 
+=begin metadata
+
+Name: hexdump
+Description: print input as hexadecimal
+Author: Michael Mikonos
+License: 
+
+=end metadata
+
+=cut
+
+use strict;
```

Comment:
This is fine for now, but this is a change outside the scope of the pull request. When something like this is included, I have to do extra work to check that it's okay. Without this, I could have immediately approved and merged the metadata change.

It's okay to add strict and warnings, but that's part of a larger issue that needs more care. Mainly, is the program warnings free for every version of Perl since v5.008? If they aren't, we also need to fix that to have warnings enabled.